packages/typescript/src/lib/TypescriptCompiler.ts

```diff
@@ -166,7 +166,75 @@ export class TypescriptCompiler {
     }
 
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    return require(tsconfigPath).compilerOptions;
+    const { compilerOptions } = require(tsconfigPath);
+
+    return this.normalizeTSConfigCompilerOptions(compilerOptions);
+  }
+
+  private normalizeTSConfigCompilerOptions(
+    options: ts.CompilerOptions
+  ): ts.CompilerOptions {
+    if (options.moduleResolution) {
+      const moduleResolution = (
+        options.moduleResolution as unknown as string
```

Comment:
According to http://json.schemastore.org/tsconfig, `moduleResolution` is not a mandatoryField.

So, invoking `toLowerCase` could be dangerous.

Reply:
There's an `if (options.moduleResolution)` check above, so this should be safe as long as it was a valid string.